Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.5
+FROM python:3.9-slim-bullseye
+ARG SDK_VERSION
+
+RUN <<EOF
+apt update --yes
+apt upgrade --yes
+apt install --yes --no-install-recommends git
+python -m pip install -U pip wheel
+python -m pip install orquestra-sdk[ray]==${SDK_VERSION}
```

Comment:
No it won't. I have considered COPYing the repo in but wasn't sure it would be better because we will need to (at least in the beginning) create images for past SDK versions and having the SDK version as a parameter makes it more flexible. Depending on the SDK release process, we can convert to COPYing the repo. WDYT @zapatacomputing/sdk-team ?

Reply:
>  I have considered COPYing the repo in but wasn't sure it would be better because we will need to (at least in the beginning) create images for past SDK versions and having the SDK version as a parameter makes it more flexible.

That `*` in the `.dockerignore` got me wondering too but it's quite clever! 

How about something like this?
```suggestion
python -m pip install "${SDK_REQUIREMENT}"
```

It would allow us to install published versions from PyPI with `SDK_REQUIREMENT='orquestra-sdk[ray]==0.47.0'` as well as tagged, but not published yet versions like `SDK_REQUIREMENT='orquestra-sdk[ray] @ git+https://github.com/zapatacomputing/orquestra-workflow-sdk.git@v0.47.0'`

Reply:
Exactly what I'm looking for. IIUC this will also let us use development versions of SDK in CE and we'll be able to mark [the task we have for this on SDK backlog](https://zapatacomputing.atlassian.net/browse/ORQSDK-833) as completed, right?

Reply:
> IIUC this will also let us use development versions of SDK in CE and we'll be able to mark [the task we have for this on SDK backlog](https://zapatacomputing.atlassian.net/browse/ORQSDK-833) as completed, right?

```
             ┌─2──────┐
         ┌──►│Task Run│
         │   └────────┘
  ┌─1─┐  │   ┌─3──────┐
  │WDR├──┴──►│Task Run│
  └───┘      └────────┘
```

Dockerfiles from this PR serve the purpose of running tasks with custom images. CE needs a WDR image (1 on the diagram). Is it gonna be the same as the image used for running tasks (2 and 3 on the diagram)? I think think the answer is "no", but wanna make sure.

If I'm right, unfortunately https://zapatacomputing.atlassian.net/browse/ORQSDK-833 is a separate thing because it's about dev versions of WDR images.